src/main/java/org/broadinstitute/barclay/help/DefaultDocWorkUnitHandler.java

```diff
@@ -243,7 +243,7 @@ protected void addHighLevelBindings(final DocWorkUnit workUnit)
      */
     protected void addCustomBindings(final DocWorkUnit currentWorkUnit) {
         final String tagFilterPrefix = getTagPrefix();
-        Arrays.stream(currentWorkUnit.getClassDoc().inlineTags())
+        Arrays.stream(currentWorkUnit.getClassDoc().tags())
```

Comment:
@magicDGS I think your example above will work as long as there is no other text in between these two lines:

```
 * @barclay.type simple
 * @author magicDGS
```
But I think it would break if it looked like this:

```
 * @barclay.type simple
 * Other comment included in between tags
 * @author magicDGS
```

In that case, the custom @barclay.type tag's text will include everything up to  the start of the next comment: "simple * Other comment included in between tags".